backend/fuzz_tests/slack/gsoc_test.py

```
@@ -0,0 +1,43 @@
+from unittest.mock import MagicMock
+
+from django.conf import settings
+from hypothesis import given
+from hypothesis import strategies as st
+from django.test import TestCase, override_settings
+
+from apps.slack.constants import OWASP_GSOC_CHANNEL_ID
+from apps.slack.events.member_joined_channel.gsoc import gsoc_handler
+
+
+class TestGsocEventHandler:
```

Ruff (0.8.2)

Missing docstring in public class

(D101)